betalife-app/package.json

```diff
@@ -11,6 +11,8 @@
     "mdbreact": "^4.26.1",
     "popper.js": "^1.16.1",
     "react": "^16.13.1",
+    "react-date-picker": "^8.0.1",
+    "react-datepicker": "^2.16.0",
```

Comment:
Why are there two date-picker libraries?

Reply:
It's an initial design. When implementing logic, some things will definitely change to better fit the theme and flow of the whole work.

Of course there'll be a clean up of unused packages before deploy.